src/pages/content/index.vue

```diff
@@ -165,4 +187,22 @@ export default {
 		margin-top: var(--space-md);
 	}
 }
+
+.first-search {
+	&__searchbar {
+		width: 100%;
+		padding: var(--space-md) 0;
+		margin: var(--space-md) 0;
+		transform: scale(1.3) translateY(250%);
+	}
+}
+
+.fade-enter-active,
+.fade-leave-active {
+	transition: opacity 0.5s;
```

Comment:
Please use a variable for the transition time:
[`var(--duration-transition-slow)`](https://github.com/schul-cloud/nuxt-client/blob/bade10e0a51014f2a1ba2198653a12055fa46a5f/src/themes/default/styles/variables/_durations.scss#L4)
```suggestion
	transition: opacity var(--duration-transition-slow);
```